list_pinned.go

```diff
@@ -0,0 +1,133 @@
+package gotwtr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+func pinnedLists(ctx context.Context, c *client, userID string, opt ...*PinnedListsOption) (*PinnedListsResponse, error) {
+	if userID == "" {
+		return nil, errors.New("look up pinned lists: userID parameter is required")
```

Comment:
```suggestion
		return nil, errors.New("pinned lists: userID parameter is required")
```

Please fix others error message, too.